src/sshping.cxx

```diff
@@ -33,13 +40,17 @@
 #include <sys/stat.h>
 #include <sys/types.h>
 #include <time.h>
-#include <unistd.h>
 #include <vector>
 
 #if (LIBSSH_VERSION_MAJOR == 0) && (LIBSSH_VERSION_MINOR < 6)
   #error "*** libssh must be version 0.6 or later"
 #endif
 
+#if defined(_MSC_VER)
```

Comment:
Let's nest this chunk inside the `#ifdef _WIN32` above, to keep it all together, unless it causes some weird dependency thing on windoze.

Reply:
You can delete this since it is now on lines 25 - 34

Double includes can cause bugs